lib/telegraf/sidekiq.rb

```diff
@@ -56,7 +56,10 @@ def call(worker, job, queue)
         }
 
         values = {
-          retry_count: job['retry_count']
+          retry_count: job['retry_count'],
+          total_workers_count: ::Sidekiq.options[:concurrency],
```

Comment:
Is this value correct? I would suspect this to be the number of threads running in parallel on the current VM. What if we run multiple processes or one process on multiple VMs, which we both do on production?

Reply:
Is this value important at all? :thinking:

Reply:
Thanks for the review, I would try to reimplement this metrics https://github.com/yabeda-rb/yabeda-sidekiq and will wrap it in periodical exporter.